packages/grafana-ui/src/components/Card/Card.mdx

```diff
@@ -0,0 +1,282 @@
+import { Meta, Preview, Props } from "@storybook/addon-docs/blocks";
+import { Card } from "./Card";
+import { Button } from '../Button';
+import { IconButton } from '../IconButton/IconButton';
+
+export const logo = 'https://grafana.com/static/assets/img/apple-touch-icon.png'
+
+<Meta title="MDX/Card" component={Card}/>
+
+# Card
+
+## Usage
+
+### Basic
+A basic Card component expects at least a heading to be used as title. Optionally a `metadata` prop is accepted, to provide some secondary information for the card. Multiple meta data elements can be provided as an array, in which case they will be separated by a horizontal line: `|`.
+```jsx
+<Card
+  heading="Filter by name"
+  description="Filter data by query."
+/>
+```
+<Preview>
+  <Card
+    heading="Filter by name"
+    description="Filter data by query."
+  />
+</Preview>
+
+
+### Multiple metadata elements
+
+```jsx
+<Card
+  heading="Test dashboard"
+  metadata={['Folder: Test', 'Views: 100']}
+  tags={['test', 'data', 'testdata']}
+/>
+```
+
+<Preview>
+  <Card
+    heading="Test dashboard"
+    metadata={['Folder: Test', 'Views: 100']}
+    tags={['test', 'data', 'testdata']}
+    onTagClick={(tag) => console.log('clicked tag:', tag) }
```

Comment:
this onTagClick is feels very wrong to me, clear indicator this is best handled via child component that uses Tag component direct?

Reply:
Agree, switching to composing child components sounds like a good idea here.